Common/Data/Mobs/Vanilla/AncientCultistSquidhead.json

```diff
@@ -1,4 +1,5 @@
 {
   "netId": 521,
+  "friendlyName": "Ancient Cultist Squidhead",
```

Comment:
What does this friendlyName property do exactly? Concerns for not using localized names if that's relevant.

Reply:
It's actually something that is generated from the API from the online-editor. It is not something used by the mod itself but am keeping the files consistent between the two

Reply:
Oh, alright. What's it do on the API's end? Just so I'm more aware.

Reply:
It's specifically for the UI.
https://pathofterraria.com/mod-data

If you go there and click on Mob Data you can see the name being presented

Reply:
Ah, got it. Thanks!